### ISSUE TITLE
Issue wrongly closed, false negative

### ISSUE THREAD

**Issue by rlohning** [google/oss-fuzz#6237]
Hi,

oss-fuzz closed and published [36218](https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=36218) today. The issue is _not_ fixed. Re-uploading the unchanged testcase found the [same issue again](https://oss-fuzz.com/testcase-detail/5643065189072896). I assume that will soon get a new issue number.

Could you please make the old issue private again?

Cheers,
Robert

**Comment by rlohning** [comment]
Same with [37025](https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=37025) which can still [reproduced by its unminimized testcase](https://oss-fuzz.com/testcase-detail/5387021181911040).

**Comment by rlohning** [comment]
The new issues numbers are [37308](https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=37308) and [37306](https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=37306), respectively.

**Comment by asraa** [comment]
Done, but did something change in the fuzzers in that range? r202108160601:202108170610

**Comment by rlohning** [comment]
Yes, I changed the call to configure to better match what's in CXXFLAGS. Resulting compiler calls used to be like:
`clang++ ... -O1 ... -O2 ...`
now they are
`clang++ ... -O1 ... -O1 ...`

Thank you for making the issues private again, but only four hours later they were published again.

**Comment by MarcoFalke** [comment]
I believe the `Restrict-View-EditIssue` label will need to be used to avoid the bot from removing it again.

**Comment by asraa** [comment]
> I believe the Restrict-View-EditIssue label will need to be used to avoid the bot from removing it again.

I added that, but I need to make sure that it doesn't remove that label because the project is verified. If I mark it explicitly as a duplicate, I believe the date should tie into that one. Sorry about that.
